src/main/java/com/johncnstn/auth/security/DomainUserDetailsService.java

```diff
@@ -20,11 +20,11 @@
     @Override
     @Transactional(readOnly = true)
     @SuppressWarnings("")
-    public DomainUserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+    public DomainUserDetails loadUserByUsername(String username) {
```

Comment:
This throws UsernameNotFoundException have to be returned, thought.

Reply:
PMD is complaining about declaring unchecked exceptions to be thrown

Reply:
Yeah
So, you have to find how to write @SupressWarning annotation and pass the pmd check.

Reply:
Left it as it is, as per offline discussion.